docs/opentracing.asciidoc

```diff
@@ -0,0 +1,94 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/ruby[elastic.co]
+endif::[]
+
+[[opentracing]]
+=== OpenTracing bridge
+
+The Elastic APM OpenTracing bridge allows to create Elastic APM `Transactions` and `Spans`,
+using the OpenTracing API.
+In other words,
+it translates the calls to the OpenTracing API to Elastic APM and thus allows for reusing existing instrumentation.
+
+The first span of a service will be converted to an Elastic APM `Transaction`,
+subsequent spans are mapped to Elastic APM `Span`.
+
+[float]
+[[operation-modes]]
+=== Operation Modes
+
+This bridge allows for different operation modes in combination with the Elastic APM `javaagent`
```

Comment:
`javaagent`?

Reply:
Yes, change of scenery. (I copied the docs. Thanks for the heads-up!)